main/chains/feeTransformers/index.js

```diff
@@ -0,0 +1,16 @@
+const { gweiToWei } = require('../../../resources/utils')
+const { intToHex } = require('@ethereumjs/util')
+
+export default {
+  137: (gasFees) => {
+    const minimumMaxPriority = gweiToWei(30)
```

Comment:
another thing we might want to try to do is always deal with gas as bignumbers until we're ready to send, then we can convert it back to hex right at the end so we dont need to keep parsing it back and forth